REINDEXING.md

```diff
@@ -0,0 +1,261 @@
+```
+
+You may wish to run a partial-reindex to verify that the process will work as expected before embarking on a complete reindex.
+
+You can monitor a reindex in progress using Grafana at [https://monitoring.wellcomecollection.org/](https://monitoring.wellcomecollection.org/), or by looking at CloudWatch metrics in the `platform` AWS account.
+
+Non-empty DLQs will be reported in the Wellcome #wc-platform-alerts Slack channel.
+
+A reindex should take less than a working day to complete. 
+
+### Connecting the Catalogue API
+
+When we have a new index in the Elasticsearch cluster created for the pipeline when we terraformed it, we will want to make that index available to the Catalogue API cluster.
+
+#### Following an index
+
+We replicate indexes between pipeline and catalogue-api clusters using [Elasticsearch CCR](https://www.elastic.co/guide/en/elasticsearch/reference/current/xpack-ccr.html).
+
+To follow an index using the console:
+
+- You can sign into Elastic Cloud here: https://cloud.elastic.co/home
+
+  Your cluster will be created in Elastic Cloud with a name patterned after : `pipeline-MY-NEW-PIPELINE`.
+
+  ![Example Elastic Cloud dashboard](.gitbook/assets/es_dash_example.png)
+
+- The catalogue-api cluster [stack-management CCR options](https://catalogue-api.kb.eu-west-1.aws.found.io:9243/app/management/data/cross_cluster_replication/follower_indices) are available via Kibana for that cluster.
+
+  You will be able to [create a follower index](https://catalogue-api.kb.eu-west-1.aws.found.io:9243/app/management/data/cross_cluster_replication/follower_indices/add)
+
+  ![Create follower index](.gitbook/assets/create_follower_es.png)
+
+  - Pick the remote cluster created by terraform above.
+  - Create the follower indexes using the naming pattern:
+  
+    - `works-indexed-MY-NEW-PIPELINE`
+    - `images-indexed-MY-NEW-PIPELINE`
+  
+  Be sure to create follower indexes for both images & works indexes. You should name the follower indexes the same as their leader. 
+  
+#### Updating configuration 
+
+When you have a complete successful reindex you will want to present it via the Catalogue API.
+
+A new index can be referenced by updating the [`ElasticConfig` object](https://github.com/wellcomecollection/catalogue-api/blob/main/common/display/src/main/scala/weco/catalogue/display_model/ElasticConfig.scala#L15):
+
+The `indexDate` should be the one used to reference the deployment and terraformed pipeline:
+
+```scala
+object ElasticConfig {
+  // We use this to share config across API applications
+  // i.e. The API and the snapshot generator.
+  val indexDate = "MY-NEW-PIPELINE"
+
+  def apply(): ElasticConfig =
+    ElasticConfig(
+      worksIndex = Index(s"works-indexed-$indexDate"),
+      imagesIndex = Index(s"images-indexed-$indexDate")
+    )
+}
+```
+
+You will want to deploy this change through the API stage environment and allow CI to perform the usual API checks. 
```

Comment:
How do I do this?

Reply:
Same as before for catalogue-api docs about deployment.